Go term copysign (#3299)
Codecademy/docs: content/go/concepts/math-functions/terms/copysign/copysign.md

---
Title: 'Copysign()'

> Description: 'Returns a value with the magnitude of a parameter and a sign of another parameter'

Reviewer: ```suggestion
Description: 'Returns a value with the magnitude and sign of the given arguments.'
```

Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Math'
  - 'Trigonometry'
  - 'Functions'
CatalogContent:
  - 'learn-go'
  - 'paths/computer-science'
---

> The **`Copysign()`** of the `math` package is an inbuilt function that takes two parameters and returns a value containing magnitude of the first parameter and a sign of the second parameter.

Reviewer: ```suggestion
The **`Copysign()`** of the `math` package is an inbuilt function that returns a value based on the magnitude of the first argument and the sign of the second argument.
```

## Syntax

```pseudo
import "math"

CpSgnXY = math.Copysign(x, y)
```

> Where `CpSgnXY` is the value of the magnitude of `x` and the sign of `y`, it is of type `float64`.

Reviewer: ```suggestion
Where `CpSgnXY` is the value, of type `float64`, composed of the magnitude of `x` and the sign of `y`.
```

## Example

> The following Golang program calculates the magnitude of `x` and the sign of `y` and prints out the result:

Reviewer: ```suggestion
The following code demonstrates the basic implementations of `.Copysign()`:
```

```go
package main

import (
  "fmt"
  "math"
)

func main() {
  fmt.Println(math.Copysign(4.9, -1))
}
```

The output will be:

```shell
-4.9
```

## Codebyte Example

> The following example shows how the `Copysign()` function handles float64 arguments.

Reviewer: ```suggestion
The following example shows how the `Copysign()` function handles various arguments.
```
